src/DebuggingContext.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Frontkom\CommonBehatDefinitions;
+
+use Behat\Behat\Hook\Scope\AfterStepScope;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Class Debugging.
+ *
+ * Provide Behat step-definitions for actions helpful when debugging.
+ */
+class DebuggingContext extends RawDrupalContext {
+
+  /**
+   * Prints from watchdog at failed step.
+   *
+   * @AfterStep
+   */
+  public function logDumpAfterStep(AfterStepScope $scope) {
+    $failed = (99 === $scope->getTestResult()->getResultCode());
+    if ($failed) {
+      $query = \Drupal::database()
```

Comment:
Hm, maybe we should either try/catch or make sure this table exists first?

Also. I am really sorry I did not notice this before. but Drupal specifics are being placed here: https://github.com/frontkom/drupal-behat-definitions

The reasoning being that some things (like waiting for an element to be visible, or failing on JS errors) is framework agnostic, and can be used in behat tests for wordpress or even next.js if you wanted to